Source/JSONAPILinks.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  JSONAPILinks.swift
+//  Kakapo
+//
+//  Created by Joan Romano on 19/06/16.
+//  Copyright © 2016 devlucky. All rights reserved.
+//
+
+import Foundation
+
+public enum JSONAPILink: CustomSerializable {
+    case Simple(value: String)
+    case Object(href: String, meta: Serializable)
```

Comment:
Basically you have two types:

- A plain string containing the url
- A link object which can contain `href` and `meta`